syntax="proto3";

option go_package = "gdarruda.me/redis-grpc/protobufs";

message PredictionRequest {
    string id_client = 1;
}

message Prediction {
    string class_name = 1;
    float value = 2;
}

message PredictionResponse {
    repeated Prediction predictions = 1;
}

service Predictions {
    rpc GetPredictions (PredictionRequest) returns (PredictionResponse);
}